indico/modules/events/editing/notifications.py

```diff
@@ -23,19 +23,20 @@ def notify_comment(comment):
     revision = comment.revision
     editable = revision.editable
     editor = editable.editor
+    submitter = editable.revisions[0].user
```

Comment:
Is this logic correct? Take an editable for a contribution with two contributors, A and B (both submitters/authors on the contribution). A submitted the initial editable. At a later stage, a revision was submitted by B. Now someone comments on that revision. If I understand the logic correctly, B won't get a notification for that comment...